src/Umbraco.Web.BackOffice/Filters/ValidateAngularAntiForgeryTokenAttribute.cs

```diff
@@ -24,6 +24,8 @@ namespace Umbraco.Web.BackOffice.Filters
     /// </remarks>
     public sealed class ValidateAngularAntiForgeryTokenAttribute : ActionFilterAttribute
     {
+        // TODO: Either make this inherit from TypeFilter or make this just a normal IActionFilter
```

Comment:
We already have an up-for-grabs task for changing this to inherit from TypeFilter